setup.py

```diff
@@ -10,14 +10,15 @@
 setup(
     name='ctapipe_io_nectarcam',
     packages=find_packages(),
-    version='0.1',
+    version='0.1', #TODO implement versioning
```

Comment:
I can open a PR setting up the `setuptools_scm` approach we use for ctapipe and other packages, if you want

Reply:
That would be great.